.github/workflows/ci.yml

```diff
@@ -58,6 +58,57 @@ jobs:
       - name: Run extended tests
         run: bash tools/test_extended.sh
 
+  run_tests_mingw_linux_64:
+    needs: check_format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - name: Install build dependencies (Linux)
+        run: sudo apt install nasm mingw-w64
+        if: runner.os == 'Linux'
```

Comment:
This will always run on Linux, no need for this check

Reply:
done, thank you